changelog/add-3981-support-phone-number

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+New support phone and email in the admin settings page.
```

Comment:
Maybe change this to `New support phone and email settings`?

Reply:
I slightly update it https://github.com/Automattic/woocommerce-payments/pull/5471/commits/b3adffb30463c52d069e2adb3508feb19e83298a 

Actually, these fields have been already in the `Card Readers` page so I want to clarify them in the admin settings page.

Reply:
I understand, the message looks great now! 👌